### PR TITLE
MTDSA-16043: Remove redundant feature switches

### PR DESCRIPTION
**Pull request by SamuelJacobsHMRC**

### CODE REVIEW

File: app/routing/VersionRoutingRequestHandler.scala [hunk 1]
```diff
@@ -42,32 +42,45 @@ class VersionRoutingRequestHandler @Inject() (versionRoutingMap: VersionRoutingM
       filters = filters.filters
     ) {
 
-  private val featureSwitch = FeatureSwitches(config.featureSwitches)
-
   private val unsupportedVersionAction = action(Results.NotFound(Json.toJson(UnsupportedVersionError)))
```
Comment (KobithasanVasantharajah) [inline app/routing/VersionRoutingRequestHandler.scala:45]:
```suggestion
  private val unsupportedVersionAction = action(Results.NotFound(UnsupportedVersionError.asJson))
```

File: app/routing/VersionRoutingRequestHandler.scala [hunk 2]
```diff
@@ -42,32 +42,45 @@ class VersionRoutingRequestHandler @Inject() (versionRoutingMap: VersionRoutingM
...
 
+  private val resourceNotFoundAction = action(Results.NotFound(NotFoundError.asJson))
+
   private val invalidAcceptHeaderError = action(Results.NotAcceptable(Json.toJson(InvalidAcceptHeaderError)))
```
Comment (KobithasanVasantharajah) [inline app/routing/VersionRoutingRequestHandler.scala:49]:
```suggestion
  private val invalidAcceptHeaderError = action(Results.NotAcceptable(InvalidAcceptHeaderError.asJson))
```

File: app/routing/Versions.scala [hunk 3]
```diff
@@ -42,18 +42,22 @@ object Version {
...
 case object Version1 extends Version {
-  val name = "1.0"
+  val name       = "1.0"
+  val configName = "1"
 }
 
 case object Version2 extends Version {
-  val name = "2.0"
+  val name       = "2.0"
+  val configName = "2"
```
Comment (KobithasanVasantharajah) [inline app/routing/Versions.scala:60]:
```suggestion
  val configName = "2"
  override val maybePrevious: Option[Version] = Some(Version1)
```

File: conf/application.conf [hunk 4]
```diff
@@ -93,12 +93,12 @@ play.modules.enabled += "config.DIModule"
 api {
   # The status of the version of the API for the API Platform.
   1.0 {
-    status = "ALPHA"
-    endpoints.enabled = false
+    status = "BETA"
+    endpoints.enabled = true
```
Comment (KobithasanVasantharajah) [inline conf/application.conf:97]: From confluence "V1 is to remain as false in production for the duration of the deprecation. " do we need to switch this to false?

**Review by KobithasanVasantharajah — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 4 above)
Comment (SamuelJacobsHMRC) [reply conf/application.conf:97]: No because these values don't affect ET or Prod

**Review by KobithasanVasantharajah — APPROVED** [review]

**Pull request merged**